dojo/templates/base.html

```diff
@@ -1062,73 +1060,67 @@ <h3 class="no-margin-top" style="padding-bottom: 5px;">
         <script src="{% static "dojo/js/cvss_calculator.js" %}"></script>
         <script type="application/javascript">
             $(document).ready(function() {
-                disable_count = document.getElementById('disable_count').innerHTML
-                refresh = document.getElementById('alert_refresh').innerHTML
-
                 $('.has-popover').popover({'trigger':'hover'});
-                {% if request.user.is_authenticated %}
+                {% if request.user.is_authenticated and not 'DISABLE_ALERT_COUNTER'|setting_enabled %}
+                    function get_alerts() {
+                        $('.dropdown-alerts').html('<div class="text-center"><i class="fa fa-spin fa-spinner"></i></div>');
```

Comment:
Please change any references of old FontAwesome icon syntax to the newer syntax (use ```fa-solid``` rather than ```fa```)

```suggestion
                        $('.dropdown-alerts').html('<div class="text-center"><i class="fa-solid fa-spinner fa-spin"></i></div>');
```